test/common.sh

```diff
@@ -0,0 +1,357 @@
+}
+
+# $1 - postgres database name.
+function create_db() {
+  local DATABASE=$1
+
+  # Create DB
+  docker exec -it $POSTGRES_CONTAINER psql -Ualgorand -c "create database $DATABASE"
+}
+
+# $1 - postgres database name.
+# $2 - pg_dump file to import into the database.
+function initialize_db() {
+  local DATABASE=$1
+  local DUMPFILE=$2
+  print_alert "Initializing database ($DATABASE) with $DUMPFILE"
+
+  # load some data into it.
+  create_db $DATABASE
+  #docker exec -i $POSTGRES_CONTAINER psql -Ualgorand -c "\\l"
+  docker exec -i $POSTGRES_CONTAINER psql -Ualgorand -d $DATABASE < $DUMPFILE > /dev/null 2>&1
+}
+
+function cleanup() {
+  kill_container $POSTGRES_CONTAINER
+  rm $CURL_TEMPFILE > /dev/null 2>&1 || true
+  kill_indexer
+}
+
+#####################
+## User Interaction #
+#####################
+
+# Interactive yes/no prompt
+function ask () {
+    # https://djm.me/ask
+    local prompt default reply
+
+    if [ "${2:-}" = "Y" ]; then
+        prompt="Y/n"
+        default=Y
+    elif [ "${2:-}" = "N" ]; then
+        prompt="y/N"
+        default=N
+    else
+        prompt="y/n"
+        default=
+    fi
+
+    while true; do
+
+        # Ask the question (not using "read -p" as it uses stderr not stdout)
+        echo -n "$1 [$prompt] "
+
+        # Read the answer (use /dev/tty in case stdin is redirected from somewhere else)
+        read reply </dev/tty
+
+        # Default?
+        if [ -z "$reply" ]; then
+            reply=$default
+        fi
+
+        # Check if the reply is valid
+        case "$reply" in
+            Y*|y*) return 0 ;;
+            N*|n*) return 1 ;;
+        esac
+
+    done
+}
+
+############################################################################
+## Integration tests are sometimes useful to run after a migration as well #
+############################################################################
+function cumulative_rewards_tests() {
+    call_and_verify 'Ensure migration updated specific account rewards.' '/v2/accounts/FZPGVIFCMHCE2HC2LEDD7IZQLKZVHRV5PENSD26Y2AOS3OWCYMKTY33UXI' 200 '"rewards":80000539878'
+    call_and_verify 'Ensure migration updated specific account rewards @ round = 810.' '/v2/accounts/FZPGVIFCMHCE2HC2LEDD7IZQLKZVHRV5PENSD26Y2AOS3OWCYMKTY33UXI?round=810' 200 '"rewards":80000539878'
+    call_and_verify 'Ensure migration updated specific account rewards @ round = 800.' '/v2/accounts/FZPGVIFCMHCE2HC2LEDD7IZQLKZVHRV5PENSD26Y2AOS3OWCYMKTY33UXI?round=800' 200 '"rewards":68000335902'
+    call_and_verify 'Ensure migration updated specific account rewards @ round = 500.' '/v2/accounts/FZPGVIFCMHCE2HC2LEDD7IZQLKZVHRV5PENSD26Y2AOS3OWCYMKTY33UXI?round=500' 200 '"rewards":28000055972'
+    call_and_verify 'Ensure migration updated specific account rewards @ round = 100.' '/v2/accounts/FZPGVIFCMHCE2HC2LEDD7IZQLKZVHRV5PENSD26Y2AOS3OWCYMKTY33UXI?round=100' 200 '"rewards":7999999996'
+}
+
+# $1 - the DB to query
+function create_delete_tests() {
```

Comment:
Here are the new tests, I run in two contexts to test the normal import, and the migration.
1) I took an e2edata archive snapshot with your script, and run these tests after setting up a DB with `algorand-indexer import`
2) I took the same e2edata archive and setup a DB using the current 2.2.1 release, then took a `pg_dump` of the database. I initialize a database with the dump, then run the migration.